capture/config_tcat.php

```diff
@@ -0,0 +1,90 @@
+<?php
+$config_file = '../config/config.json';
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
```

Comment:
This branch of the if doesn't check if config.json already exists or if the user is an admin, so you can call it to change the data even if it has already been set

Reply:
Great catch!